### PR TITLE
Multicard dragging #5

### PR DESCRIPTION
**Pull request by derrickbeining**

### CODE REVIEW

File: client/components/Stack/StackBig.js [hunk 1]
```diff
@@ -3,9 +3,8 @@ import {connect} from 'react-redux';
 import {Stack} from '../../components';
 import {
   flip3ForPlayer,
-  restartBigStackForPlayer
+  // restartBigStackForPlayer // being buggy
```
Comment (sjhelba) [inline client/components/Stack/StackBig.js:6]: I'm happy just getting rid of this restart button functionality permanently if you guys are

**Review by sjhelba — COMMENTED** [review]
Looks great and functions great-- I'm happy approving this, but I want Jess to have a chance to look through it first because it's a lot of work. There's a bug I noticed where we're able to drag cards from the field to our player stacks. I'll add this in as a separate issue on the Waffle board. Great work.

File: client/components/Stack/StackBig.js (hunk 1 above)
Comment (jess010) [reply client/components/Stack/StackBig.js:6]: Sounds good to me.

**Review by jess010 — APPROVED** [review]
I'm good with these changes.

**Pull request merged**